packages/serverless-orchestration/src/ServerlessSpoke.js

```diff
@@ -86,70 +85,31 @@ spoke.post("/", async (req, res) => {
       message: "Process exited with error 🚨",
       childProcessIdentifier: _getChildProcessIdentifier(req),
       jsonBody: req.body,
-      execResponse: execResponse instanceof Error ? execResponse.message : execResponse,
+      error: execResponse instanceof Error ? execResponse.message : execResponse,
     });
     await delay(waitForLoggerDelay); // Wait a few seconds to be sure the the winston logs are processed upstream.
     res.status(500).send({
       message: "Process exited with error",
       childProcessIdentifier: _getChildProcessIdentifier(req),
-      execResponse: execResponse instanceof Error ? execResponse.message : execResponse,
+      error: execResponse instanceof Error ? execResponse.message : execResponse,
     });
   }
 });
 
-function _execShellCommand(cmd, inputEnv, strategyRunnerSpoke = false) {
+function _execShellCommand(cmd, inputEnv) {
   return new Promise((resolve) => {
     const { stdout, stderr } = exec(
       cmd,
       { env: { ...process.env, ...inputEnv }, stdio: "inherit", maxBuffer },
```

Comment:
@mrice32 `maxBuffer` might be interesting here. We currently run w/ 8MB; I could imagine that we exceed that on occasion when the logs get spammy.

```
The maxBuffer option specifies the largest number of bytes allowed on stdout or stderr. If this value is exceeded, then the child process is terminated. This impacts output that includes multibyte character encodings such as UTF-8 or UTF-16. For instance, console.log('中文测试') will send 13 UTF-8 encoded bytes to stdout although there are only 4 characters.
```

https://nodejs.org/docs/latest/api/child_process.html#maxbuffer-and-unicode